.github/workflows/docs.yml

```diff
@@ -22,29 +22,13 @@ jobs:
           override: true
       - working-directory: gir
         run: cargo build --release
-      - run: ./gir/target/release/gir -c ./atk/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./gdk/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./gdk-pixbuf/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./gdkx11/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./gio/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./graphene/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./gtk/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./pango/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
-      - run: ./gir/target/release/gir -c ./pangocairo/Gir.toml -d ./gir-files --doc-target-path docs.md -m doc
+      - run: rustup component add rustfmt
```

Comment:
You can also use `components: rustfmt` in `actions-rs/toolchain@v1`, bit cleaner IMO (cc @bilelmoussaoui, iirc it's the same command in gtk4-rs).

Reply:
@MarijnS95 ah, i didn't knew that was possible :p I will update those some other day when I need to fix something on the CI

Reply:
Ping @GuillaumeGomez ^^

Reply:
@MarijnS95 Please send another PR for that. ;)